geoips/plugins/modules/procflows/single_source.py

```diff
@@ -797,11 +843,17 @@ def process_sectored_data_output(
 ):
     """Process sectored data output.
 
-    If product family is 'sectored_xarray_dict_to_output_format', call
+    If current product family requires a sectored dictionary of xarrays, does
+    not apply an algorithm, and DOES require an area definition, call
     'process_xarray_dict_to_output_format', store the result in a list, and return it.
     """
     output_products = []
-    if prod_plugin.family == "sectored_xarray_dict_to_output_format":
```

Comment:
Tried to remove the explicit references to individual product families throughout the code